### PR TITLE
Bookplates/fixes qa round 2

### PR DESCRIPTION
**Pull request by emb03**
@z3cka trying again, plz review

### CODE REVIEW

**Comment by emb03** [comment]
Yes, I got it.

File: www/sites/all/modules/features/uclalib_bookplates/uclalib_bookplates.module [hunk 1]
```diff
@@ -167,7 +167,7 @@ function uclalib_bookplates_menu() {
     'description' => 'Spak URL and arguments',
     'page callback' => 'drupal_get_form',
     'page arguments' => array('uclalib_bookplates_admin'),
```
Comment (z3cka) [inline www/sites/all/modules/features/uclalib_bookplates/uclalib_bookplates.module]: I wonder if this line is needed...

File: www/sites/all/modules/features/uclalib_bookplates/uclalib_bookplates.info [hunk 2]
```diff
@@ -1,7 +1,7 @@
 name = UCLALIB Bookplates
 core = 7.x
 package = UCLALIB
-version = 7.x-1.0-beta2
+version = 7.x-1.0-alpha2
```
Comment (z3cka) [inline www/sites/all/modules/features/uclalib_bookplates/uclalib_bookplates.info]: this should be `beta3`

**Comment by z3cka** [comment]
and yes, this needs to be squashed, or just re-done in one commit.

File: www/sites/all/modules/features/uclalib_bookplates/uclalib_bookplates.module (hunk 1 above)
Comment (z3cka) [inline www/sites/all/modules/features/uclalib_bookplates/uclalib_bookplates.module]: confirmed, this is needed. thanks @emb03

**Pull request closed without merging**